test/models/questionnaire_test.rb

```diff
@@ -402,6 +409,13 @@ class QuestionnaireTest < ActiveSupport::TestCase
       assert_nil questionnaire.checked_in_by_id
     end
 
+    should "return no one if user who checked-in questionnaire is deleted" do
```

Comment:
nit: Maybe should be `return nil if user...` to be consistent with previous test wording

Reply:
Which previous test? I was going initially put nil but the previous test is
`should "return no one if not checked in" do
      questionnaire = create(:questionnaire)
      assert_nil questionnaire.checked_in_by
      assert_nil questionnaire.checked_in_by_id
    end`

Reply:
In reference to the other test you added in this commit, sorry! not the previous in the file.
```return nil if author deleted```
`

Reply:
Ah gotcha, good grab! Patched